opte/src/engine/port.rs

```diff
@@ -258,15 +258,16 @@ impl PortBuilder {
 /// PortBuilder::create() -> Ready
 /// PortBuilder::restore() -> Restored
 ///
-/// Ready -- mc_start() --> Running
+/// Ready -- Port::start() --> Running
```

Comment:
I don't quite understand what these functions refer to. Are this a mapping from method name to the state that it puts the port in? I'm guessing it's really:

```
current_state -- transition_func --> new_state
```

But a comment explaining would be really helpful!

Reply:
Yea, you got it right. I'm adding text to help clarify.